src/Tizen.NUI.Components/Controls/Switch.cs

```diff
@@ -378,7 +406,7 @@ public ImageView GetCurrentTrack(SwitchExtension extension)
         [EditorBrowsable(EditorBrowsableState.Never)]
         public ImageView GetCurrentThumb(SwitchExtension extension)
         {
-            return (extension == Extension) ? Thumb : null;
+            return /*(extension == Extension) ? Thumb :*/ null;
```

Comment:
Hello, 
please remove /* xxx */

Reply:
OK

Reply:
Please remvoe both API `GetCurrentTrack` and `GetCurrentThumb`.
They are not needed anymore because now the Switch provides getter of Track and Thumb.